### PR TITLE
updated linked version of IoT Event Analytics

### PR DESCRIPTION
**Pull request by BMGEng**
Signed-off-by: Engler Michael <michael.engler@bosch.io>

### CODE REVIEW

File: src/edge.talent/Dockerfile.amd64 [hunk 1]
```diff
@@ -24,24 +24,27 @@ COPY iot-event-analytics ./iot-event-analytics
 
 COPY ./src/install-js-sdk.js ./src/
 
-COPY ./src/edge.talent ./src/edge.talent
+COPY ./src/edge.hal-interface-adapter ./src/edge.hal-interface-adapter
```
Comment (mariaivanova-git) [inline src/edge.talent/Dockerfile.amd64]: edge.talent replaced by hal-interface-adapter: is this deliberate?

File: src/edge.talent/Dockerfile.amd64 [hunk 2]
```diff
@@ -24,24 +24,27 @@ COPY iot-event-analytics ./iot-event-analytics
...
 
-WORKDIR /build/src/edge.talent
+WORKDIR /build/src/edge.hal-interface-adapter
```
Comment (mariaivanova-git) [inline src/edge.talent/Dockerfile.amd64]: as above

**Review by mariaivanova-git — COMMENTED** [review]
There are 3 modules in vehicle edge: 
- hal-interface
- hal-interface-adapter
- edge.talent
Why edge.talent Docker file links  point to hal-interface-adapter?

**Comment by BMGEng** [comment]
@mariaivanova-git I made a mistake, please take a look

**Review by mariaivanova-git — APPROVED** [review]

**Pull request merged**